docs/src/modules/components/ExamplesGrid.js

```diff
@@ -75,6 +76,21 @@ function layouts() {
       href: 'https://github.com/mui/mui-toolpad/tree/master/examples/graphql',
       source: 'https://github.com/mui/mui-toolpad/tree/master/examples/graphql',
     },
+    {
+      title: 'With WASM',
```

Comment:
How common is the WASM usecase?
If we have a limited number of slots for examples, eventually we might want to show just the common ones.
But for now I guess that we should show whatever we have!